test/integration/image-future/image-from-node-modules/node_modules/my-cool-image.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+Object.defineProperty(exports, '__esModule', {
+  value: true,
+})
+exports.default = MyCoolImage
+var _jsxRuntime = require('react/jsx-runtime')
+var _image = _interopRequireDefault(require('next/image'))
```

Comment:
```suggestion
var _image = _interopRequireDefault(require('next/future/image'))
```
Does this need to be updated to `next/future/image`

Reply:
I realize now that this test is failing with `next/future/image` because the experimental configuration is not passed to the context provider. I think that is acceptable for now because we don't really want npm packages published using experimental features so I skipped this test. We can enable again once the experimental config is no longer needed and we promote to stable status.